api/mw2pdf/get-cookies.sh

```diff
@@ -0,0 +1,2 @@
+export TORQUE_TOKEN=$(eval echo $(curl http://ec2-3-129-18-236.us-east-2.compute.amazonaws.com/GVLLIIA2020/api.php --cookie-jar cookies.jar --data "action=query" --data "meta=tokens" --data "format=json" --data "type=login" | jq '.query.tokens.logintoken'))
```

Comment:
This whole file seems potentially hard coded in ways we shouldn't -- going to create an issue for fixing this.